src/ThreeButtons.js

```diff
@@ -1,12 +1,53 @@
 import React from 'react';
 
 export default class ThreeButtons extends React.Component {
+  constructor() {
+    super();
```

Comment:
When calling parent's constructor we need to remember to pass the constructor's argument - `props`

```js
constructor(props) {
  super(props);
}
```

More about the constructor in React components: https://reactjs.org/docs/react-component.html#constructor